docker-compose-services/meilisearch/docker-compose.meilisearch.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3.6'
+services:
+  meilisearch:
+    container_name: ddev-${DDEV_SITENAME}-meilisearch
+    image: getmeili/meilisearch:v0.20.0
```

Comment:
Why so old version? We have v0.0.25 already.

Reply:
We need it for a Drupal integration module [search_api_meilisearch](https://www.drupal.org/project/search_api_meilisearch). The maintainers declare compatibility with meilisearch:0.20.0.